web/styles/modal.css

```diff
@@ -116,6 +116,7 @@
 
     &:disabled {
         cursor: not-allowed;
+        background-color: hsla(0, 0%, 65%, 1);
```

Comment:
Would be better to use `hsl` here, since hsla is legacy notation as per https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl and you are anyways using `1` as value for alpha.